app/models/move.rb

```diff
@@ -84,8 +84,8 @@ class Move < VersionedModel
   has_many :notifications, as: :topic, dependent: :destroy # NB: polymorphic association
   has_many :journeys, -> { default_order }, dependent: :restrict_with_exception, inverse_of: :move
   has_many :court_hearings, dependent: :restrict_with_exception
-  has_many :move_events, as: :eventable, dependent: :destroy # NB: polymorphic association
```

Comment:
This model exposed getter methods for the previous implementation of the runner. These getters are no longer used and this change makes the relational api between the two currently-implemented eventables (Journey, Move) the same. It meant that in the new `GenericEvents::Runner` we can now call `Move#events#create` and `Journey#events#create` in a consistent way and consolidate the two (tidy up coming in separate PR once this has been removed).